### ISSUE TITLE
Request for code

### ISSUE THREAD

**Issue by Zhihao-07** [imagine861/ML-REAXFF-Optimization#1]
Hi,

Could you please share the code used in the paper? We are quite interested in your work.

**Comment by imagine861** [comment]
> Hi,
> 
> Could you please share the code used in the paper? We are quite interested in your work.

of course, I will upload my code later, if you have any question please let me konw.

**Comment by Zhihao-07** [comment]
Many thanks!
